digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala

```diff
@@ -199,17 +199,17 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
   }
 
   def getAssetValidityDirection(bearing: Int): Int = {
-    bearing >= 270 || bearing < 90 match {
-      case true => AgainstDigitizing.value
-      case false => TowardsDigitizing.value
+    bearing > 270 || bearing <= 90 match {
+      case true => TowardsDigitizing.value
+      case false => AgainstDigitizing.value
     }
   }
 
   def getTrafficSignValidityDirection(assetLocation: Point, geometry: Seq[Point]): Int = {
 
     val mValue = GeometryUtils.calculateLinearReferenceFromPoint(Point(assetLocation.x, assetLocation.y, 0), geometry)
     val roadLinkPoint = GeometryUtils.calculatePointFromLinearReference(geometry, mValue)
-    val linkBearing = GeometryUtils.calculateBearing(geometry)
+    val linkBearing = GeometryUtils.calculateBearing(geometry, Some(mValue))
```

Comment:
Doesn't seem correct only pass Some(mvalue), will be the same like that
 **val linkBearing = GeometryUtils.calculateBearing(geometry)**

Reply:
Is not correct, I will apply some modification at GeometryUtils.calculateBearing to contemplate this.